### PR TITLE
Update error_aggregates DAG ownership

### PR DESCRIPTION
**Pull request by willkg**
Swaps wlachance out for wkahngreene as owner of error_aggregates after PR #2608.

### CODE REVIEW

**Review by jklukas — APPROVED** [review]

**Comment by willkg** [comment]
Mmm... I don't have `bqetl` set up. I did the bootstrap, but I'm missing something and hitting errors. I'm sure I can fix it, but this isn't something I'm going to do today.

If someone else could make these changes, that'd be great! Otherwise, I'll get to it when I get to it.

File: sql/moz-fx-data-shared-prod/telemetry_derived/error_aggregates_v1/metadata.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: |-
   Counts of various error measures aggregated across each unique
   set of dimensions.
 owners:
-  - wlachance@mozilla.com
+  - wkahngreene@mozilla.com
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/error_aggregates_v1/metadata.yaml:7]: Needed to update this in the query metadata in order to get the correct generated output.

**Comment by jklukas** [comment]
> Mmm... I don't have `bqetl` set up. I did the bootstrap, but I'm missing something and hitting errors. I'm sure I can fix it, but this isn't something I'm going to do today.
> 
> If someone else could make these changes, that'd be great! Otherwise, I'll get to it when I get to it.

I used this as an excuse to get a working bqetl environment going on my M1 Mac, which took a while. Should be good now. Feel free to merge once CI finishes, assuming everything looks good to you, @willkg

**Comment by willkg** [comment]
@jklukas Thank you for doing that! I really appreciate it!

**Pull request merged**